src/sentry/integrations/github/issues.py

```diff
@@ -51,12 +51,12 @@ def get_generic_issue_body(self, event: GroupEvent) -> str:
     def get_group_description(self, group: Group, event: Event | GroupEvent, **kwargs: Any) -> str:
         output = self.get_group_link(group, **kwargs)
 
-        if isinstance(event, GroupEvent) and event.occurrence is not None:
-            body = self.get_generic_issue_body(event)
-            output.extend([body])
-        elif group.issue_category == GroupCategory.PERFORMANCE:
+        if group.issue_category == GroupCategory.PERFORMANCE:
```

Comment:
Is this meant to revert what Dan just merged in https://github.com/getsentry/sentry/pull/47950 ? If so, there are a few more places, but why is it going back? Is it just to be safe for now?

Reply:
I still need custom rendering logic from `get_performance_issue_body` since I'm not displaying `evidence_display` directly but need to format `evidence_data`

Reply:
Are you sure we don't want to put this data in `evidence_display` as well? I understand we need it in data for custom email rendering, but if you put it in `evidence_display` it'll "just work" for all other integrations (including future ones) without any further customization for performance.

Reply:
yeah that's fair. I'm going to do that in the follow-up PR. I just don't want to block the rollout